Cannot find module '...\node_modules\@electron-forge\@electron-forge\cli\dist\electron-forge-start.js'

* [x] I have read the [contribution documentation](https://github.com/electron-userland/electron-forge/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

## Error

Trying to run debug task in Visual Studio Code fails on error:

```
C:\projects\project-name/node_modules/.bin/electron-forge-vscode-win.cmd --inspect-brk=39574 
module.js:549
    throw err;
    ^

Error: Cannot find module 'C:\projects\project-name\node_modules\@electron-forge\@electron-forge\cli\dist\electron-forge-start.js'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
```

## Repro

`.vscode/launch.json`

```
{
	"version": "0.2.0",
	"configurations": [
		{
			"type": "node",
			"request": "launch",
			"name": "Electron Main Process",
			"runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron-forge-vscode-nix",
			"windows": {
				"runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron-forge-vscode-win.cmd"
			},
			"runtimeArgs": [],
			"cwd": "${workspaceFolder}"
		}
	]
}
```

--- a/packages/api/cli/script/vscode.cmd
+++ b/packages/api/cli/script/vscode.cmd
@@ -4,6 +4,6 @@ SETLOCAL
 SET FORGE_ARGS=%*
 
 SET FORGE_ARGS=%FORGE_ARGS: =~ ~%
-node "%~dp0/../../@electron-forge/cli/dist/electron-forge-start.js" --vscode -- ~%FORGE_ARGS%~
+node "%~dp0/../../../@electron-forge/cli/dist/electron-forge-start.js" --vscode -- ~%FORGE_ARGS%~
 
 ENDLOCAL

--- a/packages/api/cli/script/vscode.sh
+++ b/packages/api/cli/script/vscode.sh
@@ -4,4 +4,4 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ARGS=$@
 ARGS=${ARGS// /\~ \~}
 
-node $DIR/../@electron-forge/cli/dist/electron-forge-start --vscode -- \~$ARGS\~
+node $DIR/../../../@electron-forge/cli/dist/electron-forge-start --vscode -- \~$ARGS\~

